charts/ceph-csi-rbd/values.yaml

```diff
@@ -248,7 +248,8 @@ provisioner:
     resources: {}
     ## For further options, check
     ## https://github.com/kubernetes-csi/external-provisioner#command-line-options
-    extraArgs: []
+    extraArgs:
+      - 'immediate-topology=false'
```

Comment:
why set it here not in the provisioner.yaml argument?

Reply:
either ways works, I'll add it to `provisioner.yaml`